.changeset/many-pugs-change.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes-backend': patch
+---
+
+Change add some lines in the script GkeClusterLocator.ts, the implementation consist in adding a piece of code starts in 58 adding some config.getString() method to find out if the app-config.yaml kubernetes config includes the type of Auth as googleServiceAccount
```

Comment:
The purpose of these changesets is not really to describe the code change to developers; they get used to generate release notes for end users (see some [advice](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#writing-changesets) in `CONTRIBUTING.md`)

Can you change this to be a description for end-users of what the new feature is and how to use it?

Reply:
Duley noted!

Added better description to being understandable for the end user.

Thank  you!